Sources/UtiliKit/General/URLComponents+Extensions.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  URLComponents+Extensions.swift
+//  UtiliKit-iOSTests
```

Comment:
Similar comment about module name not matching code location.
```suggestion
//  UtiliKit-iOS
```